Resources/doc/reference/permission_matrix.rst

```diff
@@ -0,0 +1,38 @@
+The permission matrix
+=====================
+
+Configuration
+-------------
+
+The type of form ``sonata_security_roles`` was improved to show quite the roles in matrix.
+Indeed, for every admin defined in sonata admin we have eight roles by default:
+
+    - Edit
+    - Create
+    - List
+    - Delete
+    - Show
+    - Export
+    - Operator
+    - Master
+
+The permssions are represented in two parts in the following table. The first part show all default permission for each entity. The second part show all custom defined roles from the ``security.yml`` configuration.
```

Comment:
show => shows (twice)
permission => permissions
Please linebreak at the new sentence